languagetool-language-modules/en/src/main/java/org/languagetool/language/English.java

```diff
@@ -425,7 +433,17 @@ protected int getPriorityForId(String id) {
       case "THE_THEM":                  return 1;   // prefer over TO_TWO
       case "THERE_THEIR":               return 1;   // prefer over GO_TO_HOME
       case "IT_IS_DEPENDING_ON":        return 1;   // prefer over PROGRESSIVE_VERBS
+      case "TO_NIGHT_TO_DAY":           return 1;   // prefer over TOO_JJ_TO
       case "IRREGARDLESS":              return 1;   // prefer over spell checker
+      case "ULTRA_HYPHEN":              return 1;   // prefer over EN_COMPOUND_ADJECTIVE_INTERNAL
+      case "THINK_BELIEVE_THAT":        return 1;
+      case "I_THINK_FEEL":              return -51;
+      case "KNOW_AWARE_REDO":           return -51;
```

Comment:
Maybe we can move these two to a cosmetically more pleasing place in the list 🤓

Reply:
Will do once it's been merged.